src/firefox/index.js

```diff
@@ -251,7 +251,37 @@ export async function useProfile(
     customPrefs = {},
   }: UseProfileParams = {},
 ): Promise<FirefoxProfile> {
-  const profile = new FirefoxProfile({destinationDirectory: profilePath});
+  let profile;
+  const finder = new FirefoxProfile.Finder();
+  const finderGetPath = promisify(finder.getPath.bind(finder));
+  try {
+    const dirExists = await isDirectory(profilePath);
+
+    if (dirExists) {
+      log.debug(`Copying profile directory from "${profilePath}"`);
+      const defaultProfilePath = await finderGetPath('default');
+      const defaultDevProfilePath = await finderGetPath('default-dev-edition');
```

Comment:
The default name of the Firefox DevEdition profile is `'dev-edition-default'` (instead of `'default-dev-edition'`)